swift/Workflow/Sources/SubtreeManager.swift

```diff
@@ -316,19 +316,19 @@ extension WorkflowNode.SubtreeManager {
 
             switch validationState {
             case .preparing:
-                fatalError("Sink sent an action inside `render`. Sinks are not valid until `render` has completed.")
+                fatalError("[\(WorkflowType.self)] Sink sent an action inside `render`. Sinks are not valid until `render` has completed.")
 
             case .pending:
                 validationState = .queued(event)
 
             case .queued:
-                fatalError("Action sent to pipe while already in the `queueing` state.")
+                fatalError("[\(WorkflowType.self)] Action sent to pipe while already in the `queueing` state.")
```

Comment:
This message doesn't make a lot of sense from a developer's perspective. I'm not entirely sure if we can tweak it to make sense. Ideally, this message would indicate what went wrong and what needs to be done to fix it.

Reply:
Yeah, I struggled with this one. Not sure what else to do for now…

Reply:
Agree this isn't a super helpful message, but adding the Workflow type as least improves it somewhat

Reply:
I should probably know this, but… uh, what leads to this state? Is it sending two events in the same render cycle? Or?

Reply:
Um… that's a great question. I'm not entirely sure, TBH. We synchronously force a re-render whenever an event is processed (as of e6938e1?). You can force this to happen by modifying `ConcurrencyTests.test_queuedEvents` to include a second call to `rendering.update()`, though I'm not sure how "possible" this is in normal use.

Reply:
Thanks @AquaGeek! If this is pretty-much impossible to happen in the real-world, it should be fine to keep this error message as-is.

Reply:
I don't think it's _impossible_, but it is highly unlikely.